Source/x11/XGServerWindow.m

```diff
@@ -2096,15 +2098,16 @@ - (int) nativeWindow: (void *)winref : (NSRect*)frame : (NSBackingStoreType*)typ
   *screen = XScreenNumberOfScreen(win_attributes.screen);
```

Comment:
You don't seem to use this variable and this looks wrong to me. This is not a very common case, we rarely create a GNUstep window for an existing X window, but I am rather sure you broke that code.

Reply:
I use it in Workspace's RecyclerIcon through NSWindow's initWithWindowRef: - works correctly. But you're right I need to assign this value to screen_id instead of monitor_id.

Reply:
Fixed.